wallet/concepts/smart-contracts.md

```diff
@@ -1,13 +1,13 @@
 ---
-description: Enable your dapp to interact with smart contracts.
-sidebar_position: 10
+description: Learn about interacting with smart contracts.
+sidebar_position: 7
 ---
 
-# Interact with smart contracts
+# Smart contracts
```

Comment:
Are descriptive headings more helpful? Also the other headings seem to start with an action

Reply:
This was moved to the Concepts section – pages in that section don't start with a verb.